### DISCUSSION TITLE
[New] Les pages sont optimisées pour le référencement naturel

### DISCUSSION THREAD

**Discussion by Ferreole** [cnumr/best-practices#93] [Bonnes pratiques] [answered]
Bonne   pratique #5108   
--
4, conseillé | 2, MEO standard | 2, impact moyen
Gain :    Durabilité du contenu | Optimiser une page pour le référencement naturel, c’est   adopter une écriture intégrant les codes du SEO.   L’objectif   s'assurer de la bonne indexation de la page,    favoriser le positionnement de la page pour répondre au mieux à l’intention de   recherche

**Comment by arnolem** [comment]
Améliorer le référencement de son site reviendrait à réduire l'impact d'une recherche ?

A mon avis, ce n'est pas parce qu'un site ressort à la place du votre que l'impact environnemental sera plus faible.

(Ou alors je n'ai pas compris l'axe d'amélioration)

**Comment by Ferreole** [comment]
Améliorer le référencement réduit le temps de recherche de l'internaute 
en lui proposant de suite la réponse. Cela réduit donc sa charge mentale.

Il faut préciser que cette BP provient d'une BP de sobriété éditoriale 
qui vise à réduire la charge mentale de l'internaute (entre autres)

Le 27/10/2021 à 21:38, Arnaud Lemercier a écrit :
>
> Améliorer le référencement de son site reviendrait à réduire l'impact 
> d'une recherche ?
>
> A mon avis, ce n'est pas parce qu'un site ressort à la place du votre 
> que l'impact environnemental sera plus faible.
>
> (Ou alors je n'ai pas compris l'axe d'amélioration)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/cnumr/best-practices/discussions/93#discussioncomment-1548334>, 
> or unsubscribe 
> <https://github.com/notifications/unsubscribe-auth/AWCKPMZJHQGVBRMRRETUZ5LUJBPMVANCNFSM5GX5462A>.
> Triage notifications on the go with GitHub Mobile for iOS 
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> 
> or Android 
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>. 
>
>
-- 
*Ferréole Lespinasse | Cyclop Éditorial*
 >> refonte éditoriale de site internet >> rédaction et production de 
contenus responsables >> conseil et formation en sobriété éditoriale : 
rédaction web, stratégie de contenus, écrire en langage clair

Alléger l'empreinte environnementale de sa communication ET la charge 
mentale de ses publics, c'est possible !

__*Discutons-en*__
+33 698 84 65 45
Visioconférence : cyclop-editorial.whereby.com/ferreole 
<https://cyclop-editorial.whereby.com/ferreole>

 > Pour vous accompagner dans vos projets : www.cyclop-editorial.eco 
<http://www.cyclop-editorial.eco/>
 > Pour vous inspirer : blog Cyclop Éditorial 
<https://www.cyclop-editorial.fr/blog-communication-redaction-web/>

 > Nouveau : pour tout savoir sur la sobriété éditoriale, c'est par ici 
! <https://www.sobriete-editoriale.fr/>


-- 
L'absence de virus dans ce courrier électronique a été vérifiée par le logiciel antivirus Avast.
https://www.avast.com/antivirus

**Comment by devcrafting** [comment]
Je comprends l'idée, mais je reste dubitatif sur "proposant de suite la réponse" car il y a justement une bataille à être le mieux placé pour une recherche...et au final ce n'est pas forcément les meilleures réponses qui sont en premier (sans compter beaucoup de doublons proches du copier/coller dans les premiers résultats).
Je ne suis pas spécialiste du sujet, mais c'est mon ressenti utilisateur et en tant qu'entrepreneur pour faire remonter notre contenu dans les recherches (et on actionné donc l'optimisation du SEO pour cela)

**Comment by Christophe-Cl** [comment]
L'optimisation SEO peut hélas apporter son lot de pratiques de développement contre-productives vis-à-vis de la sobriété numérique recherchée. C'est souvent un débat sans fin entre l'expert SEO et les devs. À défaut de cadrer précisément la BP, je ne vois pas son application telle qu'elle en v4.

**Comment by ACTLEM** [comment]
Il serait intéressant d'organiser un point visio (15-30mins) autour de cette proposition et en intégrant la https://github.com/cnumr/best-practices/discussions/95. L'idée serait d'avoir une proposition de pratique autour du SEO.

@Ferreole est-il possible que tu organises (via un framadate) un point pour les personnes intéressées par ce sujet stp?

**Comment by Ferreole** [comment]
Bonjour

voici le lien du sondage

https://framadate.org/1OgHwzVUQWLVJinw

Bien à vous tous


Le 05/11/2021 à 19:20, Thomas LEMAIRE a écrit :
>
> Il serait intéressant d'organiser un point visio (15-30mins) autour de 
> cette proposition et en intégrant la #95 
> <https://github.com/cnumr/best-practices/discussions/95>. L'idée 
> serait d'avoir une proposition de pratique autour du SEO.
>
> @Ferreole <https://github.com/Ferreole> est-il possible que tu 
> organises (via un framadate) un point pour les personnes intéressées 
> par ce sujet stp?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub 
> <https://github.com/cnumr/best-practices/discussions/93#discussioncomment-1596790>, 
> or unsubscribe 
> <https://github.com/notifications/unsubscribe-auth/AWCKPMYNAPBUCLP3YSIFHCDUKQN6HANCNFSM5GX5462A>.
> Triage notifications on the go with GitHub Mobile for iOS 
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> 
> or Android 
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>. 
>
>
-- 
*Congés du 2 au 8 novembre* -- *Ferréole Lespinasse | Cyclop Éditorial*
 >> refonte éditoriale de site internet >> rédaction et production de 
contenus responsables >> conseil et formation en sobriété éditoriale : 
rédaction web, stratégie de contenus, écrire en langage clair

Alléger l'empreinte environnementale de sa communication ET la charge 
mentale de ses publics, c'est possible !

__*Discutons-en*__
+33 698 84 65 45
Visioconférence : cyclop-editorial.whereby.com/ferreole 
<https://cyclop-editorial.whereby.com/ferreole>

 > Pour vous accompagner dans vos projets : www.cyclop-editorial.eco 
<http://www.cyclop-editorial.eco/>
 > Pour vous inspirer : blog Cyclop Éditorial 
<https://www.cyclop-editorial.fr/blog-communication-redaction-web/>

 > Nouveau : pour tout savoir sur la sobriété éditoriale, c'est par ici 
! <https://www.sobriete-editoriale.fr/>

-- 
L'absence de virus dans ce courrier électronique a été vérifiée par le logiciel antivirus Avast.
https://www.avast.com/antivirus

**Comment by Ferreole** [comment]
Bonjour Christophe et Thomas Lemaire, vous êtes 2 à avoir participé au 
sondage. En attente d'autres ? Pour l'instant c’est ce vendredi à 17h30 
qui semble coller. Voici le lien de visio

Visioconférence : cyclop-editorial.whereby.com/ferreole 
<https://cyclop-editorial.whereby.com/ferreole>

Bien à vous 2

Le 08/11/2021 à 08:02, Ferréole | Cyclop Éditorial a écrit :
>
> Bonjour
>
> voici le lien du sondage
>
> https://framadate.org/1OgHwzVUQWLVJinw
>
> Bien à vous tous
>
>
> Le 05/11/2021 à 19:20, Thomas LEMAIRE a écrit :
>>
>> Il serait intéressant d'organiser un point visio (15-30mins) autour 
>> de cette proposition et en intégrant la #95 
>> <https://github.com/cnumr/best-practices/discussions/95>. L'idée 
>> serait d'avoir une proposition de pratique autour du SEO.
>>
>> @Ferreole <https://github.com/Ferreole> est-il possible que tu 
>> organises (via un framadate) un point pour les personnes intéressées 
>> par ce sujet stp?
>>
>> —
>> You are receiving this because you were mentioned.
>> Reply to this email directly, view it on GitHub 
>> <https://github.com/cnumr/best-practices/discussions/93#discussioncomment-1596790>, 
>> or unsubscribe 
>> <https://github.com/notifications/unsubscribe-auth/AWCKPMYNAPBUCLP3YSIFHCDUKQN6HANCNFSM5GX5462A>.
>> Triage notifications on the go with GitHub Mobile for iOS 
>> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> 
>> or Android 
>> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>. 
>>
>>
> -- 
> *Congés du 2 au 8 novembre* -- *Ferréole Lespinasse | Cyclop Éditorial*
> >> refonte éditoriale de site internet >> rédaction et production de 
> contenus responsables >> conseil et formation en sobriété éditoriale : 
> rédaction web, stratégie de contenus, écrire en langage clair
>
> Alléger l'empreinte environnementale de sa communication ET la charge 
> mentale de ses publics, c'est possible !
>
> __*Discutons-en*__
> +33 698 84 65 45
> Visioconférence : cyclop-editorial.whereby.com/ferreole 
> <https://cyclop-editorial.whereby.com/ferreole>
>
> > Pour vous accompagner dans vos projets : www.cyclop-editorial.eco 
> <http://www.cyclop-editorial.eco/>
> > Pour vous inspirer : blog Cyclop Éditorial 
> <https://www.cyclop-editorial.fr/blog-communication-redaction-web/>
>
> > Nouveau : pour tout savoir sur la sobriété éditoriale, c'est par ici 
> ! <https://www.sobriete-editoriale.fr/>
-- 
*Ferréole Lespinasse | Cyclop Éditorial*
 > refonte éditoriale de site internet > rédaction et production de 
contenus responsables > conseil et formation en sobriété éditoriale

Respecter la charge mentale de vos publics, augmenter l’efficacité de 
vos actions de communication ET alléger l’empreinte environnementale de 
votre communication web, c'est possible !

__*Discutons-en*__
+33 698 84 65 45
Visioconférence : cyclop-editorial.whereby.com/ferreole 
<https://cyclop-editorial.whereby.com/ferreole>

 > Pour vous accompagner dans vos projets : www.cyclop-editorial.eco 
<http://www.cyclop-editorial.eco/>
 > Pour vous inspirer : blog Cyclop Éditorial 
<https://www.cyclop-editorial.fr/blog-communication-redaction-web/>

 > Nouveau : pour tout savoir sur la sobriété éditoriale, c'est par ici 
! <https://www.sobriete-editoriale.fr/>

-- 
L'absence de virus dans ce courrier électronique a été vérifiée par le logiciel antivirus Avast.
https://www.avast.com/antivirus

**Reply by arnolem** [reply]
Je fais mon max pour être présent vendredi à 17h30. SI je ne suis pas là, ne m'attendez pas

**Comment by ACTLEM** [comment, marked as answer]
Suite à la visio du 12/11 et la prise en compte de la proposition https://github.com/cnumr/best-practices/discussions/95, je me permets de mettre en résolu cette discussion.
